src/coreclr/src/vm/castableobject.cpp

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+#include "common.h"
+#include "castableobject.h"
+
+namespace
+{
+    OBJECTREF CallGetInterfaceImplementation(OBJECTREF *objPROTECTED, TypeHandle interfaceTypeHandle, BOOL throwIfNotFound)
```

Comment:
I don't know how costly `TypeHandle` copies are to make. Might be prudent to pass in by `&`.

Reply:
TypeHandle copies should be very cheap. They're just a union of pointer-sized or pointer POD structures.